curriculum/js1/2.js

```diff
@@ -7,7 +7,14 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if (a >= b && a >= c){
+    return a;
+  }
+  if (b >= a && b >= c){
+    return b;
+  }
+  else
```

Comment:
remove else.

Reply:
Is there specific reason why I should have to remove the "else"? I am just wondering if removing the else makes computer compile the code faster or is a good practice.  Thank you for your help.